src/server/rest/v1/service/CarService.ts

```diff
@@ -320,31 +297,30 @@ export default class CarService {
       Action.UPDATE, Entity.CAR, MODULE_NAME, 'handleUpdateCar');
     // Filter
     const filteredRequest = CarSecurity.filterCarUpdateRequest(req.body);
-    UtilsService.assertIdIsProvided(action, filteredRequest.id, MODULE_NAME, 'handleUpdateCompany', req.user);
     // Check
     UtilsService.checkIfCarValid(filteredRequest, req);
+    // Only Admin can create Pool Car
+    if (filteredRequest.type === CarType.POOL_CAR && !Authorizations.isAdmin(req.user)) {
```

Comment:
I think this case should be handled by the authorizations framework

Reply:
I added new auth named 'CREATE_POOL_CAR' to Admin and Site Admin

Reply:
Finally I introduced the Assert authorizations to check the content of an Updated/Created object.